### ISSUE TITLE
Why nodes disappear from the GUI

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/storjshare-gui#654]
### Versions

Replace the values below with your own:

- **GUI Version:** v6
- **Operating System:** all

### Expected Behavior

Please describe the program's expected behavior.

```
Error is catched, user is informed of the error and advised how to fix it. Or even better the GUI detects the error and decreases the allocated space automatically to solve the issue.
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
When someone sets a allocated space in the GUI (say 100% of the drive size), there is no problem, however, if someone writes data to the disk while Storj Share is running, there is no problem at first, all runs well. Then, when one restarts Storj Share, the node will be **gone**, why?, because it shoots a `Invalid storage size` error and prevents the drive from loading into the GUI. One can confirm this issue by trying to import the config file, it will return the exact same error and will not import the drive unless the allocated space is corrected.

Although this error requires manual input, the farmer should be informed about why the nodes are gone, and preferentially this error should be catched after the drives are loaded and a error should be presented to the farmer with the necessary instructions on how to solve it.

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Set one node to use 100% of the drive, start the node
2. Write other data to the disk while Storj is running.
3. Restart Storj Share (the app), node is gone -> `invalid storage size error` due to not enough space.

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot.

**Comment by RtypeStudios** [comment]
+1 I'm seeing the same behavior. Makes it hard to keep a machine up if after a restart storj wont kick back into life because of this bug.

**Comment by littleskunk** [comment]
It is easy to avoid. Don't fill your hard drive to the last byte...

**Comment by RtypeStudios** [comment]
Hi there, I'm not. I leave about 3-4 GB Gap

**Comment by littleskunk** [comment]
More than 1 farmer on the same drive? 3-4 GB Gap total or per farmer? How about the logfile?

**Comment by RtypeStudios** [comment]
Single farmer, nothing in the log files I can see seems to be normal chatter but I emptied them and ten minutes ago. It was a windows 2013 server and had an unexpected crash. Log files are going to a different drive.

**Comment by braydonf** [comment]
I don't think this is an issue should appear anymore with the updates for checking space available.

**Comment by braydonf** [comment]
Related: https://github.com/Storj/storjshare-gui/issues/730
